src/app/pages/storage/volumes/datasets/dataset-form/dataset-form.component.ts

```diff
@@ -616,20 +602,40 @@ export class DatasetFormComponent implements Formconfiguration{
         entityForm.formGroup.controls['recordsize'].setValue('INHERIT');
         }
         else {
-          this.ws.call('pool.dataset.query', [[["id", "=", this.parent]]]).subscribe((parent_dataset)=>{
+          this.ws.call('pool.dataset.query', [[["id", "=", this.parent]]]).subscribe((parent_dataset)=>{      
```

Comment:
Unintentional adding of whitespace at the end of the line?

Reply:
addressed.